compiler/cli/src/utils.rs

```diff
@@ -15,7 +15,7 @@ pub fn packages_path() -> PackagesPath {
         .find(|path| path.ends_with("target"))
         .unwrap();
     let candy_repo = target_dir.parent().unwrap();
-    PackagesPath::try_from(candy_repo.join("packages").as_path()).unwrap()
+    PackagesPath::try_from(candy_repo.join("packages").join("Core").as_path()).unwrap()
```

Comment:
The `PackagesPath` is the directory containing all packages that can be imported and not the path to a specific package like Core

```suggestion
    PackagesPath::try_from(candy_repo.join("packages").as_path()).unwrap()
```

Reply:
I thought this was a mistake. Without this change displaying the intermediate representation of core modules using the extension was broken for me. 

I will try to fix this issue in a separate PR.